### DISCUSSION TITLE
Flood LT (Main PNL & PED ) / INT LT (Dome)

### DISCUSSION THREAD

**Discussion by usrpc** [flybywiresim/aircraft#7959] [Help] [answered]
NOT WORKING ... after this patch => feat(lights): Interior and exterior lighting rework and new glass tex… (#7947)

**Comment by tracernz** [comment, marked as answer]
Try removing everything except the A32NX from your community directory and I think you’ll find it works. You likely have some kind of mod or livery that messes with A32NX files. If you could let us know which addon is causing it we can blacklist it.

**Comment by usrpc** [comment]
@tracernz 
Thanks for replying! 

Luckily I don' t have any mods INTERFERING with a32NX files. However THIS was an issue that may be caused by  a "Symbolic Link" (manually created by me in MSFS Community folder using the "mklink" command with the "/J" parameter. I have FIXED the issue by running again  the MKLINK command in a DOS prompt. Now the FLYBYWIRE' S Flood LT &  INT LT (Dome) is working fine again!!! NOTHING else needs to be done right now. :-)

Greetings from Switzerland
Paul

**Comment by usrpc** [comment]
**<--- UPDATE --->** 

BTW, this guy here called => "New Light ( enhancement taxi, landing light and other extrior"  Ver, 4.9.6 March 15, 2023) which can be downloaded from this website: https://flightsim.to/file/12071/new-light is NOT COMPATIBLE with the A32NX. If this addon PACK (z-Newlight-settinglight-FBW-A320NX-dev) is installed in the community folder it may also prevent the FLYBYWIRE' S A32NX Flood LT & INT LT (Dome)  to work properly.

However, I personally am NOT using this Mod but I have checked out its compatibility with the A32NX. And It is not Compatible with A32NX specially with this patch =>  feat(lights): Interior and exterior lighting rework and new glass tex… (https://github.com/flybywiresim/a32nx/pull/7947).

ThIs Mod https://flightsim.to/file/12071/new-light should be added to the **BLACKLIST**! And that' s it!
